src/com/com_nrf24l01.hpp

```diff
@@ -54,38 +55,40 @@ class NRF24L01 final : public ComInterface {
 
   /**
    * @brief Default constructor is not used.
-   * 
+   *
    */
   NRF24L01() = delete;
 
   /**
    * @brief Destroy the NRF24L01 object
-   * 
+   *
    */
   virtual ~NRF24L01() = default;
 
  private:
   std::unique_ptr<NRF24L01Core> nrf_;
   /**
-   * @brief This array holds addresses of all valid put data targets. 
-   * Beware: The last line may never change! This is the default address of any NRF as fallback. 
-   * See also types::PutDataTarget.
+   * @brief This array holds addresses of all valid put data targets.
+   * Beware: The last line may never change! This is the default address of any NRF as fallback.
+   * See also types::PutDataPacketTarget.
    */
-  std::array<data_pipe_address, 8> partner_drone_address_{{
+  std::array<types::data_pipe_address, 8> partner_drone_address_{{
```

Comment:
Please use either a comment in docstring why this has to be 8 or use a varfiable

Reply:
Explanation in docstring